src/test/moment/create.js

```diff
@@ -475,17 +475,26 @@ test('parsing RFC 2822', function (assert) {
     }
 });
 
+test('RFC 2822 offset [issue #3920]', function (assert) {
+    var timezoneTestString = 'Mon, 02 Jan 2017 06:00:00 -0800';
+    var timezoneTestDate = new Date(timezoneTestString);
```

Comment:
There are no guarantees in the ECMAScript spec that this kind of parse will work. Are we comfortable with this test? It *may* work, but there are no guarantees. Removing release label for now.